packages/main/src/themes/SegmentedButtonItem.css

```diff
@@ -0,0 +1,161 @@
+	font-family: var(--_ui5_button_fontFamily);
+	font-size: var(--sapFontSize);
+	text-shadow: var(--_ui5_button_text_shadow);
+	border-radius: var(--_ui5_button_border_radius);
+	cursor: pointer;
+	background-color: var(--sapButton_Background);
+	border: var(--sapButton_BorderWidth) solid var(--sapButton_BorderColor);
+	color: var(--sapButton_TextColor);
+	box-sizing: border-box;
+	white-space: nowrap;
+	overflow: hidden;
+	text-overflow: ellipsis;
+}
+
+.ui5-segmented-button-item-root {
+	min-width: inherit;
+	cursor: inherit;
+	height: 100%;
+	width: 100%;
+	box-sizing: border-box;
+	display: flex;
+	justify-content: center;
+	align-items: center;
+	outline: none;
+	padding: 0 var(--_ui5_button_base_padding);
+	position: relative;
+	background: transparent;
+	border: none;
+	color: inherit;
+	text-shadow: inherit;
+	font: inherit;
+	white-space: inherit;
+	overflow: inherit;
+	text-overflow: inherit;
+	letter-spacing: inherit;
+	word-spacing: inherit;
+	line-height: inherit;
+	-webkit-user-select: none;
+	-moz-user-select: none;
+	user-select: none;
+}
+
+:host(:not([active]):not([non-interactive]):not([_is-touch]):not([disabled]):hover),
+:host(:not([hidden]):not([disabled]).ui5_hovered) {
+	background: var(--sapButton_Hover_Background);
+	border: 1px solid var(--sapButton_Hover_BorderColor);
+	color: var(--sapButton_Hover_TextColor);
+}
+
+.ui5-segmented-button-item-icon {
+	color: inherit;
+	flex-shrink: 0;
+}
+
+:host([icon-end]) .ui5-segmented-button-item-root {
+	flex-direction: row-reverse;
+}
+
+:host([icon-end]) .ui5-segmented-button-item-icon {
+	margin-inline-start: var(--_ui5_button_base_icon_margin);
+}
+
+:host([icon-only]) .ui5-segmented-button-item-root {
+	min-width: auto;
+	padding: 0;
+}
+
+:host([icon-only]) .ui5-segmented-button-item-text {
+	display: none;
+}
+
+.ui5-segmented-button-item-text {
+	outline: none;
+	position: relative;
+	white-space: inherit;
+	overflow: inherit;
+	text-overflow: inherit;
+}
+
+:host([has-icon]:not([icon-end])) .ui5-segmented-button-item-text {
+	margin-inline-start: var(--_ui5_button_base_icon_margin);
+}
+
+:host([has-icon][icon-end]) .ui5-segmented-button-item-text {
+	margin-inline-start: 0;
+}
+
+:host([disabled]) {
+	opacity: var(--sapContent_DisabledOpacity);
+	pointer-events: unset;
+	cursor: default;
+}
+
+:host([has-icon]:not([icon-only])) .ui5-segmented-button-item-text{
+	/* calculating the minimum width by removing the icon size  */
+	min-width: calc(var(--_ui5_button_base_min_width) - var(--_ui5_button_base_icon_margin) - 1rem);
+}
+:host([disabled]:active) {
+	pointer-events: none;
+}
+
+:host([desktop]:not([active])) .ui5-segmented-button-item-root:focus-within:after,
+:host(:not([active])) .ui5-segmented-button-item-root:focus-visible:after,
+:host([desktop][active]) .ui5-segmented-button-item-root:focus-within:before,
+:host([active]) .ui5-segmented-button-item-root:focus-visible:before {
+	content: "";
+	position: absolute;
+	box-sizing: border-box;
+	left: 0.0625rem;
```

Comment:
As these are all same values, we can use the `inset` CSS property, instead of `top, left, right and bottom`;